experimental/generated/csv/fields.csv

```diff
@@ -1,4 +1,1754 @@
+8.9.0+exp,true,user_agent,user_agent.os.full,keyword,extended,,Mac OS Mojave,"Operating system name, including the version or code name."
+8.9.0+exp,true,user_agent,user_agent.os.full.text,match_only_text,extended,,Mac OS Mojave,"Operating system name, including the version or code name."
+8.9.0+exp,true,user_agent,user_agent.os.kernel,keyword,extended,,4.4.0-112-generic,Operating system kernel version as a raw string.
+8.9.0+exp,true,user_agent,user_agent.os.name,keyword,extended,,Mac OS X,"Operating system name, without the version."
+8.9.0+exp,true,user_agent,user_agent.os.name.text,match_only_text,extended,,Mac OS X,"Operating system name, without the version."
+8.9.0+exp,true,user_agent,user_agent.os.platform,keyword,extended,,darwin,"Operating system platform (such centos, ubuntu, windows)."
+8.9.0+exp,true,user_agent,user_agent.os.type,keyword,extended,,macos,"Which commercial OS family (one of: linux, macos, unix, windows, ios or android)."
+8.9.0+exp,true,user_agent,user_agent.os.version,keyword,extended,,10.14.1,Operating system version as a raw string.
+8.9.0+exp,true,user_agent,user_agent.version,keyword,extended,,12.0,Version of the user agent.
+8.9.0+exp,true,vulnerability,vulnerability.category,keyword,extended,array,"[""Firewall""]",Category of a vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.classification,keyword,extended,,CVSS,Classification of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.description,keyword,extended,,"In macOS before 2.12.6, there is a vulnerability in the RPC...",Description of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.description.text,match_only_text,extended,,"In macOS before 2.12.6, there is a vulnerability in the RPC...",Description of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.enumeration,keyword,extended,,CVE,Identifier of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.id,keyword,extended,,CVE-2019-00001,ID of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.reference,keyword,extended,,https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-6111,Reference of the vulnerability.
+8.9.0+exp,true,vulnerability,vulnerability.report_id,keyword,extended,,20191018.0001,Scan identification number.
+8.9.0+exp,true,vulnerability,vulnerability.scanner.vendor,keyword,extended,,Tenable,Name of the scanner vendor.
+8.9.0+exp,true,vulnerability,vulnerability.score.base,float,extended,,5.5,Vulnerability Base score.
+8.9.0+exp,true,vulnerability,vulnerability.score.environmental,float,extended,,5.5,Vulnerability Environmental score.
+8.9.0+exp,true,vulnerability,vulnerability.score.temporal,float,extended,,,Vulnerability Temporal score.
+8.9.0+exp,true,vulnerability,vulnerability.score.version,keyword,extended,,2.0,CVSS version.
+8.9.0+exp,true,vulnerability,vulnerability.severity,keyword,extended,,Critical,Severity of the vulnerability.
+=======
```

Comment:
Looks like something weird happened

Try re-running `make`

Reply:
Looks like `======` has come up.. removed it now